book/08-customizing-git/sections/policy.asc

```diff
@@ -427,7 +427,7 @@ target_shas.each do |sha|
   end
 end
 ----
-This script uses a syntax that wasn't covered in the Revision Selection section of Chapter 6. You get a list of commits that have already been pushed up by running this:
+This script uses a syntax that wasn't covered in the <<_revision_selection>> section of <<_git_tools>>. You get a list of commits that have already been pushed up by running this:
```

Comment:
I think this will render weirdly. Let's just do this:

> This script uses a syntax that wasn't covered in <<_revision_selection>>.

I think that'll render better in all formats.